islandora_basic_collection.links.action.yml

```diff
@@ -1,3 +1,8 @@
+islandora_basic_collection.ingest_action:
```

Comment:
Should this entry get it's weight of `-1`: https://github.com/Islandora/islandora_solution_pack_collection/pull/209/files#diff-f7990cf10d52b3d4d46d8c0e9daaafa2R27 ?

Appears to be allowed for, based on the last example in https://www.drupal.org/docs/8/api/menu-api/providing-module-defined-local-actions

Reply:
Just to say it explicitly: probably should _not_ need the `hook_menu_local_tasks_alter()` implementation in order to apply the weights.

... at least, I hope?